create-shield-study

```diff
@@ -1,3 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env node
 
-./node_modules/.bin/plop --plopfile plop-shield-study.js $@
+const pathJoin = require("path").join;
+
+process.argv.push('--plopfile', pathJoin(__dirname, "plop-shield-study.js"));
+require("plop");
```

Comment:
I switched from bash to hopefully a more cross-OS friendly Node script.
This fixes #4; "command not found: create-shield-study"

I was testing locally against my PR branch using the following sauce:

```sh
$ npx pdehaan/create-shield-study#general-fixes fancy-new-study
```